sample/.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
+node_js: "node"
+
+script: npm start
```

Comment:
Where is the command to run the automation test? If you miss it, please add it to here.